lib/srv/db/proxyserver.go

```diff
@@ -516,6 +516,16 @@ func (s *ProxyServer) Proxy(ctx context.Context, proxyCtx *common.ProxyContext,
 		return trace.Wrap(err)
 	}
 
+	var labels prometheus.Labels
+	if len(proxyCtx.Servers) > 0 {
+		labels = getLabelsFromDB(proxyCtx.Servers[0].GetDatabase())
+	} else {
+		labels = getLabelsFromDB(nil)
+	}
```

Comment:
I don't think it's possible to have empty `proxyCtx.Servers` here. but good to check anyway xD